LICENSE

```diff
@@ -0,0 +1,13 @@
+ Copyright 2017 ReproNim
```

Comment:
is there some kind of a ReproNim entity which could hold the copyright?

I would go for "ReproNim Developers" but even then not sure e.g. who is holding the copyright for work done by TCG folks -- might be TCG.